src/content/utils.ts

```diff
@@ -36,15 +37,37 @@ export async function extractSection(element: Element) {
 }
 
 export async function findCourseInfo(code: string) {
+  let requestOptions: RequestInit
+  let headers: Headers
+
   const urlencoded = new URLSearchParams()
   urlencoded.append("q", code)
+  urlencoded.append("clientRequestID", generateRandomHexId())
```

Comment:
use [`randomUUID`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) to generate this properly instead - your custom function can probably be removed.